docs/lob_guide/src/asciidoc/_chapters/working_with_lob.adoc

```diff
@@ -751,4 +751,380 @@ CQD `LOB_MAX_SIZE` (default 10G expressed in M [10000M]).
+  , DESCSYSKEY                       LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKNUM                         INT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , CHUNKLEN                         LARGEINT NO DEFAULT NOT NULL NOT DROPPABLE SERIALIZED
+  , DATAOFFSET                       LARGEINT DEFAULT NULL SERIALIZED
+  , STRINGPARAM                      VARCHAR(400) CHARACTER SET ISO88591 COLLATE DEFAULT DEFAULT NULL SERIALIZED
+  , PRIMARY KEY (DESCPARTNKEY ASC, DESCSYSKEY ASC, CHUNKNUM ASC)
+  )
+  SALT USING 8 PARTITIONS
+;
+
+--- SQL operation complete.
+```
+
+[#get statement for lob tables]
+== Get Statement for LOB Tables
+
+The GET Statement for lob tables gives column specific information on any lob columns in a table. 
+
+[#syntax]
+=== Syntax
+
+```
+GET LOB STATS FOR TABLE table-name;
+SELECT * FROM TABLE(LOB STATS table-name);
+```
+
+[#examples]
+=== Examples
+
+Suppose the following tables have been created:
+
+```
+>>CREATE TABLE tlob130gt (c1 INT NOT NULL, c2 BLOB, c3 CLOB, c4 BLOB, PRIMARY KEY (c1));
+
+--- SQL operation complete.
+
+>>CREATE TABLE tlob130gt2 (c1 INT NOT NULL, c2 BLOB, c3 CLOB, c4 BLOB STORAGE 'EXTERNAL', PRIMARY KEY (c1));
+
+--- SQL operation complete.
+```
+
+* This Get Statement displays statistics for the table tlob130gt, the lob information is formatted for machine readability.
+
++
+
+```
+>>SELECT left(trim(catalog_name) || '.' || trim(schema_name) || '.' || trim(object_name) , 20), left(trim(column_name),5), left(trim(lob_location),5) , left(trim(lob_data_file),20),LOB_DATA_FILE_SIZE_EOD,LOB_DATA_FILE_SIZE_USED from table(lob stats(tlob130gt));
+
+(EXPR)      (EXPR)     (EXPR)     (EXPR)     LOB_DATA_FILE_SIZE_EOD    LOB_DATA_FILE_SIZE_USED
+------      ------     ------     ------     ----------------------    ----------------------
+
+TRAFODION.SCH.TLOB13   C2   /user/trafodion/lobs   LOBP_044744252290302   15   10
+TRAFODION.SCH.TLOB13   C3   /user/trafodion/lobs   LOBP_044744252290302   15   10
+TRAFODION.SCH.TLOB13   C4   /user/trafodion/lobs   LOBP_044744252290302   45   30
+
+--- 3 row(s) selected.
+```
+
+
+* This Get Statement displays statistics for the table tlob130gt2, the lob information is formatted for human readability.
+
++
+
+```
+>>GET LOB STATS FOR TABLE tlob130gt2;
+
+Lob Information for table: "TRAFODION".LOB130.TLOB130GT2
+=========================
+
+ColumnName:  C2
+Lob Location:  /user/trafodion/lobs
+LOB Data File:  LOBP_07468755986685501835_0001
+LOB EOD:  0
+LOB Used Len:  0
+ColumnName:  C3
+Lob Location:  /user/trafodion/lobs
+LOB Data File:  LOBP_07468755986685501835_0002
+LOB EOD:  0
+LOB UsedLen:  0
+ColumnName:  C4
+Lob Location:  External HDFS Location
+LOB Data File:  External HDFS File
+LOB EOD:  0
+LOB Used Len:  0
+
+--- SQL operation complete.
+```
+
+* This Get Statement displays statistics for the table tlob130gt2, the lob information is formatted for machine readability.

```

Comment:
Select Statement?

Reply:
Thanks Dave.
I've rewritten this part. :)